libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1365,6 +1363,9 @@ export class RushConfiguration {
    * Does this project have subspaces
    */
   public get hasSubspaces(): boolean {
+    if (!this._projects) {
```

Comment:
The `_cachedRushProjectsBySubspaceName` name is still confusing. A "cache" isn't generally guaranteed to be a complete dataset. Caches are used for improving efficiency by deduplicating work. If this is guaranteed to be complete after initialization has completed, it probably shouldn't be called a "cache."

Reply:
Renamed to _rushProjectsBySubspaceName